projects/packages/publicize/src/class-publicize.php

```diff
@@ -262,6 +279,8 @@ public function get_all_connections_for_user() {
 									'connection_id'  => $connection['connection_data']['id'],
 									'can_disconnect' => current_user_can( 'edit_others_posts' ) || get_current_user_id() === $user_id,
 									'profile_link'   => $this->get_profile_link( $service_name, $connection ),
+									'shared'         => $connection['connection_data']['user_id'] === '0',
+									'status'         => 'ok',
```

Comment:
I'm not sure we should return this property unless it's requested. It might not be 'ok'

Reply:
`'ok'` unless proven that it is `broken`?

Reply:
Yeah, the default state is 'ok'